docs/rfcs/component-universal-telemetry.md

```diff
@@ -0,0 +1,212 @@
+# Pipeline Component Telemetry
+
+## Motivation and Scope
+
+The collector should be observable and this must naturally include observability of its pipeline components. Pipeline components
+are those components of the collector which directly interact with data, specifically receivers, processors, exporters, and connectors.
+
+It is understood that each _type_ (`filelog`, `batch`, etc) of component may emit telemetry describing its internal workings,
+and that these internally derived signals may vary greatly based on the concerns and maturity of each component. Naturally
+though, there is much we can do to normalize the telemetry emitted from and about pipeline components.
+
+Two major challenges in pursuit of broadly normalized telemetry are (1) consistent attributes, and (2) automatic capture.
+
+This RFC represents an evolving consensus about the desired end state of component telemetry. It does _not_ claim
+to describe the final state of all component telemetry, but rather seeks to document some specific aspects. It proposes a set of
+attributes which are both necessary and sufficient to identify components and their instances. It also articulates one specific
+mechanism by which some telemetry can be automatically captured. Finally, it describes some specific metrics and logs which should
+be automatically captured for each kind of pipeline component.
+
+## Goals
+
+1. Define attributes that are (A) specific enough to describe individual component[_instances_](https://github.com/open-telemetry/opentelemetry-collector/issues/10534)
+   and (B) consistent enough for correlation across signals.
+2. Articulate a mechanism which enables us to _automatically_ capture telemetry from _all pipeline components_.
+3. Define specific metrics for each kind of pipeline component.
+4. Define specific logs for all kinds of pipeline component.
+
+## Attributes
+
+All signals should use the following attributes:
+
+### Receivers
+
+- `otel.component.kind`: `receiver`
```

Comment:
given the kind is now in the measurement name, do we still need this attribute?

Reply:
Yes. Only metrics have names, so we need this in order to be able to correlate with other signals. Even the metrics should still have the attribute so that they can be correlated.